pkg/sdk/v1beta1/kafkacluster_types.go

```diff
@@ -104,6 +104,9 @@ type BrokerConfig struct {
 	Tolerations        []corev1.Toleration           `json:"tolerations,omitempty"`
 	KafkaHeapOpts      string                        `json:"kafkaHeapOpts,omitempty"`
 	KafkaJVMPerfOpts   string                        `json:"kafkaJvmPerfOpts,omitempty"`
+	// With the use of this config broker log4jConfig can be reconfigured, if left empty
+	// the default config will be used
```

Comment:
This comment is a little bit messy for me, maybe something like "Override for the default log4j configuration" or something like that.